performance-tests/bench/report_parser/ArgumentParser.cpp

```diff
@@ -0,0 +1,133 @@
+#include "ArgumentParser.h"
+
+#include <util.h>
+#include <json_conversion.h>
+#include <BenchTypeSupportImpl.h>
+
+#include <unordered_map>
+
+using namespace Bench;
+
+bool ArgumentParser::parse(int argc, ACE_TCHAR* argv[], OutputType& output_type,
+  OutputFormat& output_format, Report& report, std::ofstream& output_file_stream)
+{
+  std::string input_file_path;
+  std::string output_file_path;
+
+  if (argc == 1) {
+    show_usage_prompt();
+  } else {
+    try {
+      for (int i = 1; i < argc; i++) {
+        const ACE_TCHAR* argument = argv[i];
+        std::string option_argument = get_option_argument(i, argc, argv);
+
+        if (!ACE_OS::strcmp(argument, ACE_TEXT("--help"))
```

Comment:
When I run this locally, I get "Option --help requires an argument". If I supply an argument it displays the message I'd expect.

Reply:
I just found that myself.  I need to change that code to pull out the option argument after checking the argument for "-h" or "--help".